src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx

```diff
@@ -325,4 +329,110 @@ describe('MobileView component', () => {
       expect(mockHandleAssigneeDetails).toBeCalledTimes(1);
     });
   });
+
+  it('Test that mark as paid button, renders: adjust amount , assignee name, amount set in Sats and next button', async () => {
+    const props: CodingBountiesProps = {
+      ...defaultProps,
+      paid: false
+    };
+
+    uiStore.setMeInfo({
+      ...user,
+      owner_alias: props.person.owner_alias
+    });
+
+    render(<MobileView {...props} />);
+
+    const inputSAT = screen.findByTestId('input_sats');
+    const nextBTN = screen.findByTestId('next_btn');
+
+    expect(screen.queryByText('Adjust the amount')).toBeInTheDocument();
+    expect(screen.queryByText('Guest Developer')).toBeInTheDocument();
+    expect(screen.queryByTestId('USDText')).toBeInTheDocument();
+    expect(screen.queryByText('SAT')).toBeInTheDocument();
+  });
+
+  it('Test that increment and decrement button on input field for amount works correctly', async () => {
+    const props: CodingBountiesProps = {
+      ...defaultProps,
+      paid: false
+    };
+
+    uiStore.setMeInfo({
+      ...user,
+      owner_alias: props.person.owner_alias
+    });
+
+    render(<MobileView {...props} />);
+
+    const inputSAT = screen.getByTestId('input_sats');
+
+    expect(inputSAT).toBeInTheDocument();
+
+    fireEvent.change(inputSAT, { target: { value: 100 } });
+
+    await waitFor(() => expect(inputSAT).toHaveValue(100), { timeout: 3000 });
+  });
+
+  it('Test that clicking on next button takes you to Award Badge section assert all the necessary ui elements', async () => {
+    const props: CodingBountiesProps = {
+      ...defaultProps,
+      creatorStep: 2,
+      paid: false
+    };
+
+    uiStore.setMeInfo({
+      ...user,
+      owner_alias: props.person.owner_alias
+    });
+
+    render(<MobileView {...props} />);
+    expect(screen.getByText('Mark as Paid')).toBeInTheDocument();
+    expect(screen.queryByText('Award Badge')).toBeInTheDocument();
+    expect(screen.getAllByTestId('check_box').length).toBe(2);
+  });
+
+  it('Test that button state and text changes according to selection in Award Badge Section', () => {
+    const props: CodingBountiesProps = {
+      ...defaultProps,
+      creatorStep: 2,
+      paid: false
+    };
+
+    uiStore.setMeInfo({
+      ...user,
+      owner_alias: props.person.owner_alias
+    });
+
+    render(<MobileView {...props} />);
+
+    const checkBox = screen.getAllByTestId('check_box');
+
+    fireEvent.click(checkBox[0]);
+
+    (async () => {
+      await waitFor(() => {
+        expect(screen.getByText('Mark as Paid'));
+      });
+    })();
+  });
+
+  it('Test that if sufficient funds are available, bounty is paid', async () => {
```

Comment:
@okhot This does not prove that Payment  action can be taken if the organization has sufficient funds

Reply:
I've created a bounty and tried the Payout flow and the payment always goes through with or without sufficient funds. That's what got me stuck there.

Reply:
> I've created a bounty and tried the Payout flow and the payment always goes through with or without sufficient funds. That's what got me stuck there.

Are you getting this error on the Pay Bounty button, or the Mark As Paid Button?

Reply:
![Screenshot 2024-02-16 at 18 55 53](https://github.com/stakwork/sphinx-tribes-frontend/assets/33989423/f17ee10c-8d74-48fe-bfdc-103e59af84f7)

Reply:
> ![Screenshot 2024-02-16 at 18 55 53](https://private-user-images.githubusercontent.com/33989423/305509079-f17ee10c-8d74-48fe-bfdc-103e59af84f7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgxMTA5OTYsIm5iZiI6MTcwODExMDY5NiwicGF0aCI6Ii8zMzk4OTQyMy8zMDU1MDkwNzktZjE3ZWUxMGMtOGQ3NC00OGZlLWJmZGMtMTAzZTU5YWY4NGY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE2VDE5MTEzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzZWMzODBiM2E5NGE0OTUzY2YzN2MyNTM3ZjBiNmE4MzQ2YzdhZjVhYmUyOTI3MmE3NTYwOGI2MDhiOWEyNjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6-2cwhLd1Z-_DDegb-rxBpAaogpEZytfNCPWAcg2Gno)

oh! Yeah, the Mark as Paid button will always be displayed,  test that the mark as paid button updates the bounty paid state, and I will update the button test ticket, thanks.